app/models/audit.rb

```diff
@@ -17,5 +17,7 @@ class Audit < ApplicationRecord
     uploaded: "uploaded",
     started: "started",
     withdrawn: "withdrawn",
+    document_invalidated: "document_invalid",
+    document_changed_to_validated: "document_changed_to_valid",
```

Comment:
everything else in here the keys and values are the same, but they're different here, and I'm not sure why?

Reply:
it was the artist writer in me 😆  I'll change them to match